templates/rapidsai-core/Base.dockerfile.j2

```diff
@@ -42,7 +42,11 @@ COPY packages.sh /opt/docker/bin/
 {# Cleaup conda and set ACLs for all users #}
 {% include 'partials/cleanup-chmod.dockerfile.j2' %}
 
-RUN apt-get update && apt-get -y upgrade
+{% if "centos" in os %}
+RUN yum -y update && yum -y upgrade
+{% else %}
+RUN apt -y update && apt -y upgrade
+{% endif %}
```

Comment:
Looks like this same code is duplicated across the `Base`, `Runtime`, and `Devel` templates, so it'd likely be a good candidate for a Jinja partial. Can you add it to the `partials/patch.dockerfile.j2` file? This is already included in all 3 image types and was made particularly for patching CVEs.